### PR TITLE
Add support for configuring the AJAX call error function.

### PR DESCRIPTION
**Pull request by gcotelli**

### CODE REVIEW

File: source/Willow.package/WillowJavascriptNamespace.class/instance/setHandleAjaxErrorCallFunction..st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Configuring
+setHandleAjaxErrorCallFunction: aJSFunction
```
Comment (mtabacman) [inline source/Willow.package/WillowJavascriptNamespace.class/instance/setHandleAjaxErrorCallFunction..st]: Rename method to one of the following:
a) setHandleAjaxErrorCallFunctionTo:
b) handleAjaxErrorsWith:
c) handleAjaxErrorsUsing:

File: source/Willow.package/WillowNamespaceFileMetadataLibrary.class/instance/jswillowJsContent.st [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+var Willow;
+
+if (Willow === undefined) {
+    Willow = {
+        version: "4.0.0",
+        handleAjaxErrorCall: function (xmlHttpRequest, textStatus, errorThrown) {
+        }
+    };
+};
+' expandMacrosWith: WillowConstants handleAjaxErrorCallFunctionName
```
Comment (mtabacman) [inline source/Willow.package/WillowNamespaceFileMetadataLibrary.class/instance/jswillowJsContent.st]: It seems handleAjaxErrorCallFunctionName is not used in the content string.

Comment (gcotelli) [inline source/Willow.package/WillowNamespaceFileMetadataLibrary.class/instance/jswillowJsContent.st]: Fixed in the next commit

File: source/Willow.package/WillowJavascriptNamespace.class/instance/setHandleAjaxErrorCallFunction..st (hunk 1 above)
Comment (gcotelli) [inline source/Willow.package/WillowJavascriptNamespace.class/instance/setHandleAjaxErrorCallFunction..st]: Fixed in the next commit

**Pull request merged**